app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -0,0 +1,185 @@
+package com.kickstarter.viewmodels
+
+import android.util.Pair
+import androidx.annotation.NonNull
+import com.kickstarter.libs.ActivityRequestCodes
+import com.kickstarter.libs.Environment
+import com.kickstarter.libs.FragmentViewModel
+import com.kickstarter.libs.rx.transformers.Transformers.*
+import com.kickstarter.libs.utils.BooleanUtils
+import com.kickstarter.libs.utils.DateTimeUtils
+import com.kickstarter.models.Project
+import com.kickstarter.models.Reward
+import com.kickstarter.models.StoredCard
+import com.kickstarter.ui.ArgumentsKey
+import com.kickstarter.ui.data.ActivityResult
+import com.kickstarter.ui.data.ScreenLocation
+import com.kickstarter.ui.fragments.PledgeFragment
+import rx.Observable
+import rx.subjects.BehaviorSubject
+import rx.subjects.PublishSubject
+import java.math.RoundingMode
+
+interface PledgeFragmentViewModel {
+    interface Inputs {
+        /** Call when user deselects a card they want to pledge with. */
+        fun closeCardButtonClicked(position: Int)
+
+        /** Call when the new card button is clicked. */
+        fun newCardButtonClicked()
```

Comment:
Just curious, but do y'all use the term `clicked ` a lot? We tend to use `tapped` 🤔

Reply:
Yeah we're clicked people 😬but I think it's because in android you `addOnClickListener`